main/util.py

```diff
@@ -188,6 +188,12 @@ def update_query_argument(name, value=None, ignore='cursor', is_list=False):
   return '%s%s' % (flask.request.path, '?%s' % query if query else '')
 
 
+def parse_tags(tags, separator=None):
```

Comment:
Yes, there is quite an _import fest_ :)

in the good old days we didn't import `config` or `utils` in the [model.py](https://github.com/gae-init/gae-init/blob/0.14.2/main/model.py#L13)